bin/opteadm/build.rs

```diff
@@ -0,0 +1,31 @@
+use std::error::Error;
+use std::process::Command;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    println!("cargo:rerun-if-changed=../../.git");
+
+    let commit_count_out =
```

Comment:
Perhaps we could consider also adding the commit SHA from `HEAD` itself? I've found that helpful in many situations. The `vergen` and `built` crates provide easy access to a number of pieces of build information, such as timestamps, which may also be helpful.